configs/4.0/ffigen.yaml

```diff
@@ -3,19 +3,18 @@ preamble: |
   // Use of this source code is governed by a BSD-style license that can be
   // found in the LICENSE file.
 
-  // ignore_for_file: camel_case_types, non_constant_identifier_names
-  // ignore_for_file: unused_element, unused_field
+  // ignore_for_file: type=lint, unused_element, unused_field
```

Comment:
https://github.com/dart-lang/ffigen/pull/532/files
According to this patch, "type=lint" doesn't needs to be written here, as it is added automatically.

Reply:
> Automatically generate `ignore_for_file: type=lint` **if not specified in preamble.**

It is possible to specify only `// ignore_for_file: unused_element, unused_field` in the preamble, but the output will not look very clean. Try running ffigen without `type=lint` once and you'll see what I mean.

Reply:
> Try running ffigen without type=lint once and you'll see what I mean.

I know that  two 'ignore_for_file' is created.
(I tested ffigen/example/libclang-example)
```
// ignore_for_file: camel_case_types, non_constant_identifier_names

// AUTO GENERATED FILE, DO NOT EDIT.
//
// Generated by `package:ffigen`.
// ignore_for_file: type=lint

```
 but I don't think it's a problem that two 'ignore_for_file' are written because it is generated file(I don't think user need to think about the code style of the generated files. personal opinion hahaha....). And 'type=lint' is because I think it works like a default(?) option. Anyway it is not important things. Let's merge it.